test/libdrampowertest/lib_test.cc

```diff
@@ -82,6 +82,21 @@ int main(int argc, char* argv[])
         // Need at least tWRAPDEN = AL + CWL + BL/2 + WR + 1 cycles between WR and PDN_F_PRE
         test.doCommand(MemCommand::PDN_F_PRE,3,265);
 
+        // Exit from Precharge Power-down
+        test.doCommand(MemCommand::PUP_PRE,3,300);
+        // Activate bank 0
+        test.doCommand(MemCommand::ACT,0,350);
+        // Precharge all banks with bank 0 active
+        test.doCommand(MemCommand::PREA,0,400);
+        // Precharge all banks again
+        test.doCommand(MemCommand::PREA,0,450);
+        // Activate bank 0 twice
```

Comment:
maybe add here that a warning will be generated and that this is for testing the warning.
Anyway felipe will do a new library test that will test almost everything.